Makefile

```diff
@@ -46,6 +52,9 @@ fpga-test:
 	make clean setup fpga-run BOARD=AES-KU040-DB-G INIT_MEM=1 USE_EXTMEM=0 
 	make clean setup fpga-run BOARD=AES-KU040-DB-G INIT_MEM=0 USE_EXTMEM=1 
 
+syn-run: 
+	make syn-build SYN=1 INIT_MEM=0 USE_EXTMEM=0 
```

Comment:
If you are calling the syn makefile you should not need the variable

Reply:
i thought it was to mimick how the fpga and sim-test work

Reply:
there's no SIM or FPGA variable....